scanner/laravel.go

```diff
@@ -96,56 +96,63 @@ func LaravelCallback(appName string, srcInfo *SourceInfo, plan *plan.LaunchPlan)
 	}
 
 	// generate Dockerfile if it doesn't already exist
+	dockerfileExists := true
 	_, err = os.Stat("Dockerfile")
 	if errors.Is(err, fs.ErrNotExist) {
+		dockerfileExists = false
+	}
```

Comment:
Do you want to handle other types of errors here (e.g. permissions errors)? Or just delegate that to the Dockerfile generator later?

Reply:
I'm not that worried about that case.  Delegated to even later.... If the Dockerfile exits and can't be read, the generator will skip over it and deploy will later fail, and to be honest I'm OK with that... it is no different than if your source files were unreadable.

Put another way, the biggest thing I'm concerned about is us not stomping over existing Dockerfiles.  If you provide one, and it is broken, then that's your problem.  We will provide you with the tools to regenerate a new one, but how you proceed at that point is entirely up to you.